overlay-test/parts/overlay.html

```diff
@@ -0,0 +1 @@
+<!-- wp:pattern {"slug":"overlay-test/overlay-1"} /-->
```

Comment:
This is not showing up yet as the default overlay, it's expecting a user created one